src/main/resources/config/config-template.conf

```diff
@@ -0,0 +1,26 @@
+input.osm: {
+  #@optional
+  pbf: {
+    file: String
+  }
+}
+input.asset: {
+  #@optional
+  file: {
+    directory: String
+    hierarchic = Boolean | false
+  }
+}
+output: {
+  #@optional
+  file: {
+    directory: String
+    hierarchic: Boolean | false
+  }
+}
+generation: {
+  #@optional
+  lv: {
+    distinctHouseConnections: Boolean | false
```

Comment:
what does this parameter do? Please add documentation behind it.

Reply:
Is added in rtd.. :eyes: